ios/RNCCameraRoll.mm

```diff
@@ -715,6 +714,18 @@ static void RCTResolvePromise(RCTPromiseResolveBlock resolve,
     return mediaSubTypeLabels;
 }
 
+- (NSArray<NSString *> *) getAlbumsForAsset:(PHAsset *)asset {
+    NSMutableArray<NSString *> *albumTitles = [NSMutableArray array];
+
+    PHFetchResult<PHAssetCollection *> *collections = [PHAssetCollection fetchAssetCollectionsContainingAsset:asset withType:PHAssetCollectionTypeAlbum options:nil];
```

Comment:
It looks like this is very slow. Maybe this feature should be off by default?

Reply:
Agreed. I see no way to speed up this. So behind an include flag is a good idea. Will work on it.

Reply:
Updated! @Arjan-Zuidema What do you think? In this version flag only change something in iOS because Android is not impacted.

Reply:
Thanks! But I'd rather keep the api's the same for Android and iOS if possible to minimize confusion. Can you also make it to not return the albums on Android when the include flag is not set?

Reply:
*2 ~days~ months later...* I updated the code to return albums only if the include flag is set @Arjan-Zuidema